test/OrchardCoreContrib.PoExtractor.Tests/Fakes/FakeStringLocalizer.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace OrchardCoreContrib.PoExtractor.Tests.Fakes
+{
+    public class FakeStringLocalizer
```

Comment:
Why it doesn't implement `IPluralizeStringLocalizer`?

Reply:
Do you mean [IPluralStringLocalizer](https://github.com/OrchardCMS/OrchardCore/blob/a845c2e5c3a2a445a6a534cb984b3ad2f7708253/src/OrchardCore/OrchardCore.Localization.Abstractions/IPluralStringLocalizer.cs#L8)?

I didn't want to add a package reference to OrchardCore, and the extractor never looks at the type of the localizer. It only looks for variables that are named S, H. or T, regardless of what the type of the variable is.

Reply:
> only looks for variables that are named S, H. or T, regardless of what the type of the variable is.

Right, but no problem if we add a reference to the `OC.Localization.Abstractions`

Reply:
I changed it to use `IStringLocalizer` instead of `FakeStringLocalizer`. I don't see a CI triggered by my commit though. Or a test run.

Reply:
If you never mind, I modified the unit tests